myth

```diff
@@ -177,147 +170,126 @@ if args.solv:
-                exitWithError(args.outform, "Error syncing database over IPC: " + str(e))
-            except ConnectionError as e:
-                exitWithError(args.outform, "Could not connect to RPC server. Make sure that your node is running and that RPC parameters are set correctly.")               
+    elif args.init_db:
+        try:
+            contract_storage.initialize(eth, args.sync_all)
+        except FileNotFoundError as e:
+            exitWithError(args.outform, "Error syncing database over IPC: " + str(e))
+        except ConnectionError as e:
+            exitWithError(args.outform, "Could not connect to RPC server. Make sure that your node is running and that RPC parameters are set correctly.")
 
     sys.exit()
 
-
 # Load / compile input contracts
 
 contracts = []
+address = None
 
-if (args.code):
+if args.code:
     address = util.get_indexed_address(0)
-
     contracts.append(ETHContract(args.code, name="MAIN"))
 
 # Get bytecode from a contract address
 
-elif (args.address):
-
+elif args.address:
     address = args.address
-
     if not re.match(r'0x[a-fA-F0-9]{40}', args.address):
         exitWithError(args.outform, "Invalid contract address. Expected format is '0x...'.")
 
     try:
         code = eth.eth_getCode(args.address)
-
-        if (code == "0x"):
-            exitWithError(args.outform, "Received an empty response from eth_getCode. Check the contract address and verify that you are on the correct chain.") 
-
     except FileNotFoundError as e:
         exitWithError(args.outform, "IPC error: " + str(e))
     except ConnectionError as e:
         exitWithError(args.outform, "Could not connect to RPC server. Make sure that your node is running and that RPC parameters are set correctly.")
     except Exception as e:
         exitWithError(args.outform, "IPC / RPC error: " + str(e))
-
-    contracts.append(ETHContract(code, name=args.address))
+    else:
+        if code == "0x":
+            exitWithError(args.outform, "Received an empty response from eth_getCode. Check the contract address and verify that you are on the correct chain.")
+        else:
+            contracts.append(ETHContract(code, name=args.address))
 
 # Compile Solidity source file(s)
 
-elif (len(args.solidity_file)):
-
+elif args.solidity_file:
     address = util.get_indexed_address(0)
-
-    if(args.graph and len(args.solidity_file) > 1):
+    if args.graph and len(args.solidity_file) > 1:
         exitWithError(args.outform, "Cannot generate call graphs from multiple input files. Please do it one at a time.")
 
     for file in args.solidity_file:
-
         if ":" in file:
             file, contract_name = file.split(":")
         else:
             contract_name = None
 
-        file = file.replace("~", str(Path.home()))  # Expand user path
+        file = os.path.expanduser(file)
 
         try:
-            signatures.add_signatures_from_file(file, sigs)  # Parse file for new function signatures
```

Comment:
I also removed some unnecessary comments if the variable or function name is clear enough